helm-chart/values.yaml

```
@@ -0,0 +1,84 @@
+# Default values for json-rpc-relay.
+
+replicaCount: 1
+
+image:
+  repository: #IMAGE is not published yet, use internal repo/location to push to
```

It's published, you can pull 
```suggestion
  repository: ghcr.io/hashgraph/hedera-json-rpc-relay:main
```